doc/controller_configuration/controller_configuration_tutorial.rst

```diff
@@ -30,146 +79,407 @@ The ``controllers.yaml`` configuration file is located in the ``robot_moveit_con
 
-Now, you should be ready to have MoveIt talk to your robot.
+Multi Controller Manager
+------------------------
 
-Debugging Information
----------------------
-The ``FollowJointTrajectory`` or ``GripperCommand`` interfaces on your robot must be communicating in the namespace: ``/name/action_ns``. In the above example, you should be able to see the following topics (using *rostopic list*) on your robot:
+The *MoveIt ROS Control Controller Manager* can only interface controllers from the single ROS controller manager found in the ROS namespace defined by the ROS parameter ``~ros_control_namespace`` (defaults to ``/``).
 
- * /panda_arm_controller/follow_joint_trajectory/goal
- * /panda_arm_controller/follow_joint_trajectory/feedback
- * /panda_arm_controller/follow_joint_trajectory/result
- * /hand_controller/gripper_action/goal
- * /hand_controller/gripper_action/feedback
- * /hand_controller/gripper_action/result
+To overcome this limitation, there also exists ``MoveItMultiControllerManager``, which queries *all* existing ROS controller managers and instantiates all controllers with their respective namespace taking care of proper delegation.
+This type of manager can be configured by setting ``moveit_controller_manager`` to ``moveit_ros_control_interface::MoveItMultiControllerManager``:
 
-You should also be able to see (using ``rostopic info topic_name``) that the topics are published/subscribed to by the controllers on your robot and also by the **move_group** node.
+.. code-block:: XML
 
-Remapping /joint_states topic
------------------------------
+    <param name="moveit_controller_manager" value="moveit_ros_control_interface::MoveItMultiControllerManager" />
 
-When you run a `move group node <../move_group_interface/move_group_interface_tutorial.html>`_, you may need to remap the topic /joint_states to /robot/joint_states, otherwise MoveIt won't have feedback from the joints. To do this remapping you could make a simple launch file for your node as follows: ::
+.. note::
+  The `Simple Controller Manager` is generic as it can interface controllers from multiple ROS controller managers by specifying different prefix names in the bridging configuration file.
 
-  <node pkg="moveit_ros_move_group" type="move_group" name="any_name" output="screen">
-    <remap from="joint_states" to="robot/joint_states"/>
-  </node>
+Controller Switching and Namespaces
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Or you can make a subscriber with the correct topic name and then ensure that the starting robot state for your move group corresponds to a correct joints angle by using the call back of this subscriber.
```

Comment:
I explicitly moved that section to the end as it is not strongly related to the current sections.